packages/db/prisma/schema.prisma

```diff
@@ -213,9 +213,9 @@ model Payment {
     customerId          String
     membershipPlan      String
     membership          Membership
-    membershipStartDate Int? // Shouldn't these be DateTime?
-    membershipEndDate   Int?
-    canceledDate        Int?
+    membershipStartDate DateTime?
+    membershipEndDate   DateTime?
+    canceledDate        DateTime?
```

Comment:
what's the difference with canceledDate and subscriptionCanceledDate?

Reply:
They are the same fields but kept in different tables. In user table we have it for easy access. In payment table we have it for logs. Like you may have 3 different subscription objects in the payment table but the user table will always store the latest